benchexec/tools/esbmc.py

```diff
@@ -31,13 +31,19 @@ def version(self, executable):
     def name(self):
         return "ESBMC"
 
-    def cmdline(self, executable, options, tasks, propertyfile, rlimits):
-        assert len(tasks) == 1, "only one inputfile supported"
-        inputfile = tasks[0]
-        return [executable] + ["-p", propertyfile] + options + [inputfile]
+    def cmdline(self, executable, options, task, rlimits):
+        data_model_param = get_data_model_from_task(task, {"ILP32": "32", "LP64": "64"})
+        if data_model_param and "--arch" not in options:
+            options += ["--arch", data_model_param]
+        return (
+            [executable]
+            + ["-p", task.property_file]
+            + options
+            + [task.single_input_file]
+        )
 
-    def determine_result(self, returncode, returnsignal, output, isTimeout):
-        output = "\n".join(output)
+    def determine_result(self, run):
+        output = "\n".join(run.output)
```

Comment:
Here is another case for using `run.output.text` (without additional blank lines) or `any_line_contains` (which actually seems easy to use here).

Reply:
The tool developers use a function that they have defined themselves.
Although I understand that this could be made better but in general, I refrain from code cleanup enthusiasm for the piece of code that does not belong to me.

That was the reason I have kept it so, and also some other cases like this e.g., above comment in symbiotic.

Reply:
fixed in 4cf9ec8e54d8a90a314ac265d5cbbbdb589e8851